lib/generators/solidus_stripe/install/templates/config/initializers/solidus_stripe.rb

```diff
@@ -11,6 +11,10 @@
   # Number of seconds while a webhook event is valid after its creation.
   # Defaults to the same value as Stripe's default.
   # config.webhook_signature_tolerance = 150
+  #
+  # Name of the `Spree::RefundReason` used for Stripe-generated refunds.
```

Comment:
I'd add some lines to explain what to do when changing preference.

Reply:
Done!

> If you change it, make sure that the corresponding `Spree::RefundReason` exists in the database with that name.